spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Fusion.scala

```diff
@@ -167,11 +196,15 @@ private[mkldnn] object Fusion {
       weight.mul(alpha)
 
       val bias = convBias.storage().array()(j)
-      val mean = originMean.storage().array()(j)
+      val mean = originMean.storage().array()(j) / bn.scaleFactor.storage().array()(0)
```

Comment:
no need to div scaleFactor, as mean is the real running mean.

Reply:
OK. I will delete them.